core/src/main/java/com/scalar/db/api/Update.java

```diff
@@ -0,0 +1,124 @@
+package com.scalar.db.api;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableMap;
+import com.scalar.db.io.Column;
+import com.scalar.db.io.Key;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import javax.annotation.concurrent.NotThreadSafe;
+
+/** A command to update an entry to a storage. */
```

Comment:
Assuming that we're referring to an entry that exists in of the underlying storage, I think we should use `in` instead of `of`.

```suggestion
/** A command to update an entry in the underlying storage. */
```